admin/google_search_console/class-gsc-service.php

```diff
@@ -79,8 +84,11 @@ public function get_crawl_issue_counts() {
 		$crawl_error_counts = $this->get_crawl_error_counts( $this->profile );
 
 		$return = array();
+		// Ignore coding standards for object properties.
+		// @codingStandardsIgnoreStart
```

Comment:
Those properties should pass as long as they are on the whitelist as added in https://github.com/Yoast/wordpress-seo/pull/6875/commits/14f775fc7a7342dbf14e42a93870bf457b679d36